src/main/java/com/sprint/mission/discodeit/controller/ReadStatusController.java

```diff
@@ -0,0 +1,44 @@
+package com.sprint.mission.discodeit.controller;
+
+import com.sprint.mission.discodeit.dto.ReadStatusCreateRequest;
+import com.sprint.mission.discodeit.dto.ReadStatusUpdateRequest;
+import com.sprint.mission.discodeit.entity.ReadStatus;
+import com.sprint.mission.discodeit.service.ReadStatusService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/api/readStatus")
```

Comment:
url에는 보통 카멜케이스보다는 `-`(하이픈)을 주로 사용해요.  
```suggestion
@RequestMapping("/api/read-status")
```